zipline/assets/asset_writer.py

```diff
@@ -165,6 +165,62 @@ def _generate_output_dataframe(data_subset, defaults):
     return output
 
 
+# Define a version number for the database generated by these writers
+# Increment this version number any time a breaking change is made to the
+# schema and readers of the database
+ASSET_DB_VERSION = 1
```

Comment:
Since we're calling the SQLite file `assets.db`, I suggest we call this variable `ASSETS_DB_VERSION`. Also, how about starting at 0.